U&Dave

Lexzie opened the pull request:
According to Logopedia this should be the colors for this logo.
I use light logos and the '&' was too dark in the former logo. This should be better for both light and dark backgrounds I think.

Lexzie committed:
U&Dave

According to Logopedia this should be the colors for this logo

Lexzie committed:
Merge branch 'master' into U&Dave

wedebe commented:
I've added a comment to a previous PR, along with official svgs modded to use official colours :)

https://github.com/picons/picons/pull/2289#issuecomment-2231924502

Lexzie commented:
> I've added a comment to a previous PR, along with official svgs modded to use official colours :)
> 

Yes and I added an answer that we already have the official colouring.
